code/game/mob/living/carbon/human/species/species_attack.dm

```diff
@@ -8,8 +8,6 @@
 /datum/unarmed_attack/claws
 	attack_verb = list("scratched", "clawed", "slashed")
 	attack_noun = list("claws")
-	eye_attack_text = "claws"
```

Comment:
Why???

Reply:
We're not adding mobs with claws to be kill-grabbing eye-squishing humans anytime soon, and this code is shitty.